### ISSUE TITLE
capacity.threefoldtoken website don't reflect the correct the zos version of nodes

### ISSUE THREAD

**Issue by hossnys** [threefoldtech/jumpscale_lib#137]
#### Detailed description
capacity.threefoldtoken website don't reflect the correct the z-os version of nodes , it shows it  as v.1.4.1 when node is development as shown in attached images
![screenshot from 2018-09-17 15-01-08](https://user-images.githubusercontent.com/26298009/45624661-3fe51b80-ba8b-11e8-903b-89bbec3c0fd3.png)
![screenshot from 2018-09-17 15-00-32](https://user-images.githubusercontent.com/26298009/45624662-407db200-ba8b-11e8-8cbc-a86d20a14068.png)

**Comment by hossnys** [comment]
they are the same node because as you see the Mac of ens3 interface in first screenshot is the same node id in the second one , but the scenario that happened for this node is that it was installed with z-os 1.4.1 then i shutdown it and install again with z-os development ,but the os version didn't update the value correctly

**Comment by zaibon** [comment]
yeah but that's exactly why I asked, cause the node id is supposed to be the mac address of the interface that has the gateway so it should be the mac of the ens3 interface.

**Comment by maxux** [comment]
What's the status of this ?
